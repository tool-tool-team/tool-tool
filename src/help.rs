use crate::{NAME, VERSION, DESCRIPTION};

pub fn print_help() {
    println!("{} {}", NAME, VERSION);
    println!();
    println!("{}", DESCRIPTION);
    println!();
}
