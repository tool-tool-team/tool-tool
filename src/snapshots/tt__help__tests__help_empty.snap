---
source: src/help.rs
expression: help_text
---
"ðŸ”§ tool-tool 0.1.0 ðŸ”§\n\nðŸ”§ A light-weight meta-tool to version and install tool dependencies for your software projects\n\n\nUsage: tt [-v] <command> <args...>\n  Run tool <command> with the provided arguments\n\nFlags:\n  -v     Verbose debug output\n\nNo tool-tool file named .tool-tool.v1.yaml found in current directory or ancestors, please create one and configure your tools.\nRefer to https://gitlab.com/manuelwoelker/tool-tool/-/blob/master/README.md for further information\n\nUsage: tt --download\n  Download configured tools for later use\n"
